app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -548,6 +549,7 @@ class PatientRepository @Inject constructor(
     return when (identifierType) {
       BpPassport -> createBpPassportMetaData(assigningUser)
       BangladeshNationalId -> createBangladeshNationalIdMetadata(assigningUser)
+      EthiopiaMedicalRecordNumber -> TODO()
```

Comment:
This commit is now broken since `TODO()` will cause a crash since the screens will end up creating a record of type `EthiopiaMedicalRecordNumber` which will be passed here.

Suggestion: We can move the enabling of the identifier for the `Country` to a separate, final commit so that we can build this independently without needing to worry about this.